src/Pact/Analyze/Translate.hs

```diff
@@ -1508,7 +1508,9 @@ translateNode astNode = withAstContext astNode $ case astNode of
     Some ty tm' <- translateNode tm
     pure $ Some SStr $ CoreTerm $ Typeof ty tm'
 
-  AST_NFun node "yield" [ obj ] -> do
+  -- NOTE: we ignore the optional target chain during analysis, for now at
```

Comment:
Isn't this kind of gross? The function could have 3 arguments

Reply:
FV currently assumes that typechecking succeeds before analysis, so we should be safe here.

Reply:
I should clarify -- typechecking *is* run before FV analysis.